core/src/main/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListener.java

```diff
@@ -51,12 +54,14 @@
  * latency (average, 90th, 99th, etc).
  */
 public class MetricsResourceMethodApplicationListener implements ApplicationEventListener {
+
+  public static final String RUNTIME_TAGS_PROP_KEY = "_request_tags";
```

Comment:
where does this come from? Since we're getting tags out of the container request, the value here is set by Jersey? If so, is there no existing public constant from Jersey instead?

Reply:
the contant name should perhaps be REQUEST_TAGS_PROP_KEY

Reply:
@ewencp These can be added via the filters as necessary. So, basically allows you to inject tags based on the request. Let's say if someone has an authentication filter and want to track all the metrics by the user, they could do so by using this.